test/regression.uts

```diff
@@ -3574,18 +3576,18 @@ def test_show():
         tr6 = TracerouteResult6(tr6_packets)
         tr6.show()
         result = cmco.get_output()
-    expected = "  2001:db8::1                               :udpdomain \n"
-    expected += "1  2001:db8::1                                3         \n"
-    expected += "2  2001:db8::2                                3         \n"
-    expected += "3  2001:db8::3                                3         \n"
-    expected += "4  2001:db8::4                                3         \n"
-    expected += "5  2001:db8::5                                3         \n"
-    expected += "6  2001:db8::6                                3         \n"
-    expected += "7  2001:db8::7                                3         \n"
-    expected += "8  2001:db8::8                                3         \n"
-    expected += "9  2001:db8::9                                3         \n"
-    expected += "10 2001:db8::10                               3         \n"
-    expected += "11 2001:db8::11                               3         \n"
+    expected = "   2001:db8::1                               :udp0 \n"
```

Comment:
You introduced a regression. What is the goal of changing UDP ports behaviour?

Reply:
It provides bug n°2 from happening again.

For instance in this test, which has nothing to do about DNS, just because we use UDP we get a “domain” appearing.